src/utils/thread_pool_win.c

```diff
@@ -0,0 +1,230 @@
+#include <Windows.h>
```

Comment:
General comment, add more comments for how the code works, because code regarding threads is usually not the most obvious to understand

Reply:
Done